gapic-generator/templates/default/service/client/method/_def.erb

```diff
@@ -10,6 +10,7 @@
 #
 <%= render partial: "service/client/method/docs/error", locals: { method: method } -%>
 #
+<%= render partial: "service/client/method/docs/snippets", locals: { method: method } -%>
```

Comment:
my personal pref would be to have `<%- if method.generate_yardoc_snippets? -%>` here instead of in the subtemplate because then it can be seen that this part may not be rendered and the question of 'why is there no `#` after this render' does not come up.

Reply:
I think there are trade-offs. Putting the "if"s here would tend to obfuscate the structure a bit, and the question of the missing `#` will be there regardless.